modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -106,6 +106,14 @@ def buildProject(Map options = [:]) {
     )
   ]
 
+  if (options.publishingE2ETests == true && env.PUBLISHING_E2E_TESTS_BRANCH == null) {
+    parameterDefinitions << stringParam(
```

Comment:
I hadn't realised the `parameterDefinitions` were in here.  Now it's pointed out it's obvious!